templates/zabbix-agent-systemd.init.erb

```diff
@@ -1,16 +1,17 @@
 [Unit]
 Description=Zabbix Agent
-After=syslog.target network.target
-Documentation=man:zabbix_agentd
+After=syslog.target
+After=network.target
 
 [Service]
+EnvironmentFile=-/etc/sysconfig/zabbix-agent
```

Comment:
@bastelfreak Removed this line. Anyway there is no /etc/sysconfig/zabbix-agent file even on redhat.